test/run_pass/attrs.js

```diff
@@ -0,0 +1,202 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+(function() {
+  var attrs = {
+    'test_cwd.js': {
+      skip: ['all'],
+      reason: "this test had @STDOUT=COMMAND[pwd], but there's no way to " +
+      "check out current directory path with js driver. So it skips " +
+      "temoporalily."
```

Comment:
The file name is run_pass, but it seems these tests are failed. I think this is somewhat misleading.